subway_prediction_ml/test/test_subway_pipeline.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import os
+import sklearn
+import sys
+import unittest
+
+sys.path.append(os.path.abspath(os.path.join(__file__, '..', '..')))
+from subway_pipeline import SubwayPipeline
+
+
+class TestSubwayPipeline(unittest.TestCase):
```

Comment:
It would definitely be a sounder design to pass dataframes around rather than re-read CSVs all over. However I think it makes sense to have the columns be selected in the various pipeline step objects, reason being they make those modules a bit more self-documenting that way. It means using `usecols` to select the columns is out, but I bet there's 14 other ways to do it.